README.md

```diff
@@ -44,482 +32,16 @@ KubeCarrier is currently in early development and is not ready for production us
-
-```bash
-# Offering objects contain information about CRDs that are shared to a Tenant.
-# They contain all the information to validate and create new instances.
-$ kubectl get offering -n team-b --as=team-b-member
-NAME                        DISPLAY NAME   PROVIDER   AGE
-couchdbs.eu-west-1.team-a   CouchDB        team-a     3m15s
-
-# Region exposes information about the underlying Clusters.
-$ kubectl get region -n team-b --as=team-b-member
-NAME               PROVIDER   DISPLAY NAME   AGE
-eu-west-1.team-a   team-a     EU West 1      5m14s
-
-# Provider exposes information about the Provider of an Offering.
-$ kubectl get provider -n team-b --as=team-b-member
-NAME     DISPLAY NAME   AGE
-team-a   The A Team     6m11s
-```
-
-### 6. Internal and External CRD
-
-In [4. Catalog Entries](#4-catalog-entries), we created two CRDs. A public one, that users can interact with and an internal one.
-This split allows the Provider to override user properties or hide settings and status information from their users.
-
-Now we will create a `CouchDB` instance and see how we work with those objects in KubeCarrier:
-
-<details>
-<summary><b>CouchDB definition</b></summary>
-
-```yaml
-apiVersion: eu-west-1.team-a/v1alpha1
-kind: CouchDB
-metadata:
-  name: db1
-spec:
-  username: hans
-  password: hans2000
-```
-</details>
-
-```bash
-$ kubectl apply -n team-b --as=team-b-member \
-  -f https://raw.githubusercontent.com/kubermatic/kubecarrier/master/docs/manifests/couchdb.eu-west-1.yaml
-catalog.catalog.kubecarrier.io/default created
-
-# This is the object we just created as team-b
-$ kubectl get couchdb -n team-b --as=team-b-member
-NAME   USERNAME   PASSWORD   AGE
-db1    hans       hans2000   4s
-
-# There is also an internal representation, that users cannot access:
-$ kubectl get couchdbinternal -n team-b --as=team-b-member
-Error from server (Forbidden): couchdbinternals.eu-west-1.team-a is forbidden: User "team-b-member" cannot list resource "couchdbinternals" in API group "eu-west-1.team-a" in the namespace "team-b"
-
-# Only members of the Provider team (team-a) can access these objects:
-kubectl get couchdbinternal -n team-b --as=team-a-member
-NAME   USERNAME   PASSWORD   VERSION   AGE
-db1    hans       hans2000             31s
-```
-
-**Ok, what is happening here?**
-
-Team A is offering the `CouchDB` service from their Kubernetes cluster `eu-west-1` and Team B created an instance of the `CouchDB` service.
-
-Because Team A decided to hide the `.spec.version` property it's absent from the CRD that tenants of Team A have access to. While the internal CRD retains that field, so the provider can use it to orchestrate their workload.
+- [What is KubeCarrier](./docs/_index.md)
+- [Getting Started](./docs/getting_started)
+- [Architecture](./docs/architecture/_index.md)
+- [Repository Organization](./docs/repository_organization/_index.md)
+- [Glossary](./docs/glossary/_index.md)
+- [API Reference](./docs/api_reference/_index.md)
```

Comment:
I'd delete this, linking to the docs should be cleaner. See for example how the Kubernetes or etcd do it, they just link to the docs and you should go from there. They don't list docs index in the README.md

Reply:
I list docs index because kubermatic and kubeone also did that, WDYT @thetechnick ?